deploy/deploy.go

```diff
@@ -165,6 +169,8 @@ func (d *Deployment) Deploy(ctx context.Context, kubecfg string) error {
 	vOut.Reset()
 
 	d.Ingress.SetKClient(kClient)
+	d.Ingress.SetRCfg(rCfg)
```

Comment:
is there a downside to setting the mClient directly instead of rCfg? I think that would be more in line with how we set kClient, but maybe I'm missing a subtle difference